core/src/test/java/org/apache/iceberg/deletes/TestPositionFilter.java

```diff
@@ -282,6 +286,16 @@ public void testPositionSetRowFilter() {
 
   @Test
   public void testCombinedPositionSetRowFilter() {
+    testCombinedPositionSetRowFilter(null);
```

Comment:
minor: why not make the test a `@ParameterizedTest`, taking a `null` and a configured `ExecutorService`?

Reply:
Done.